contracts/TaxManager.sol

```diff
@@ -15,24 +15,19 @@ contract TaxManager is ITaxManager {
 
     address public custodian;
 
-    address public seekerTaxPool;
     address public solverTaxPool;
-    address public maintenancePool;
-    address public devPool;
-    address public rewardAllocationPool;
-    address public perpetualPool;
-    address public tierPool;
     address public revenuePool;
-    address public marketingPool;
 
-    uint256 public seekerTaxRate;
+    // Tax Receiver Addresses
+    address private referralTaxReceiver;
```

Comment:
good logic for pools
initial platform tax goes to `platformTaxReceiver`

leftovers from referral tax go to `referralTaxReceiver`

Reply:
```
// Tax Receiver Addresses
    address private referralTaxReceiver;
    address private platformTaxReceiver;
```

Why are the addresses private btw?